[2023-04-08 02:59:54,003] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2023-04-08T02:59:49.979422+00:00 [queued]>
[2023-04-08 02:59:54,012] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2023-04-08T02:59:49.979422+00:00 [queued]>
[2023-04-08 02:59:54,013] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-04-08 02:59:54,014] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-04-08 02:59:54,015] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-04-08 02:59:54,127] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): run_after_loop> on 2023-04-08 02:59:49.979422+00:00
[2023-04-08 02:59:54,132] {standard_task_runner.py:52} INFO - Started process 354 to run task
[2023-04-08 02:59:54,136] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', 'manual__2023-04-08T02:59:49.979422+00:00', '--job-id', '13', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpbtprgi_l', '--error-file', '/tmp/tmp4gi8vzvp']
[2023-04-08 02:59:54,137] {standard_task_runner.py:80} INFO - Job 13: Subtask run_after_loop
[2023-04-08 02:59:54,208] {task_command.py:369} INFO - Running <TaskInstance: example_bash_operator.run_after_loop manual__2023-04-08T02:59:49.979422+00:00 [running]> on host a51dc0b874ee
[2023-04-08 02:59:54,289] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=run_after_loop
AIRFLOW_CTX_EXECUTION_DATE=2023-04-08T02:59:49.979422+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-08T02:59:49.979422+00:00
[2023-04-08 02:59:54,291] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-04-08 02:59:54,292] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo 1']
[2023-04-08 02:59:54,312] {subprocess.py:85} INFO - Output:
[2023-04-08 02:59:54,315] {subprocess.py:92} INFO - 1
[2023-04-08 02:59:54,316] {subprocess.py:96} INFO - Command exited with return code 0
[2023-04-08 02:59:54,345] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20230408T025949, start_date=20230408T025954, end_date=20230408T025954
[2023-04-08 02:59:54,391] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-04-08 02:59:54,422] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
